src/filesystem/browser/filesystem-commands.ts

```diff
@@ -60,7 +66,8 @@ export class FileCommandContribution implements CommandContribution {
         @inject(FileSystem) protected readonly fileSystem: FileSystem,
         @inject(ClipboardService) protected readonly clipboardService: ClipboardService,
         @inject(SelectionService) protected readonly selectionService: SelectionService,
-        @inject(OpenerService) protected readonly openerService: OpenerService
+        @inject(OpenerService) protected readonly openerService: OpenerService,
+        @inject(CppClientContribution) protected readonly clientContribution: CppClientContribution
```

Comment:
Should not be here, the cpp extension can depend on the fs extension, not another way around